examples/mongocxx/get_values_from_documents.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+
+#include <bsoncxx/builder/stream/document.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/client.hpp>
+
+using namespace bsoncxx;
```

Comment:
Suggest pulling in the types you need from bsoncxx individually, instead of adding a using-directive.  This suggestion is mostly for consistency with the other examples in this directory.  Note also that the Google C++ style guide bans using-directives entirely (see <https://google.github.io/styleguide/cppguide.html#Namespaces), although it looks like the driver uses them moderately.  Mostly in test code, it seems.